public/sitemap-0.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
100% we don't need to push something like this to repo.

Reply:
Removed generated sitemaps and added them to .gitignore.

Reply:
We don't need to have them in `.gitignore`, because these files are out of scope of this task.